tests/actions.rs

```diff
@@ -0,0 +1,235 @@
+//! Alert tests
+#[macro_use]
+extern crate serial_test;
+extern crate fantoccini;
+extern crate futures_util;
+
+use crate::common::sample_page_url;
+use fantoccini::actions::{
+    Actions, InputSource, KeyAction, KeyActions, MouseActions, NullActions, PointerAction,
+    MOUSE_BUTTON_LEFT,
+};
+use fantoccini::keys::Key;
+use fantoccini::{error, Client, Locator};
+use std::time::Duration;
+use time::Instant;
+
+mod common;
+
+async fn actions_null(mut c: Client, port: u16) -> Result<(), error::CmdError> {
+    let sample_url = sample_page_url(port);
+    c.goto(&sample_url).await?;
+    let null_actions = NullActions::new("null".to_string()).pause(Duration::from_secs(1));
+    let now = Instant::now();
+    c.perform_actions(vec![null_actions].into()).await?;
+    assert!(now.elapsed().as_seconds_f64() >= 1.0);
+    Ok(())
+}
+
+async fn actions_key(mut c: Client, port: u16) -> Result<(), error::CmdError> {
+    let sample_url = sample_page_url(port);
+    c.goto(&sample_url).await?;
+
+    // Test pause.
+    let key_pause = KeyActions::new("key".to_string()).pause(Duration::from_secs(1));
+    let now = Instant::now();
+    c.perform_actions(vec![key_pause].into()).await?;
+    assert!(now.elapsed().as_seconds_f64() >= 1.0);
+
+    // Test key down/up.
+    let mut elem = c.find(Locator::Id("text-input")).await?;
+    assert_eq!(elem.prop("value").await?.unwrap(), "");
+
+    let key_actions = KeyActions::new("key".to_string())
+        .then(KeyAction::Down { value: 'a' })
+        .then(KeyAction::Up { value: 'a' });
+    elem.click().await?;
+    c.perform_actions(vec![key_actions].into()).await?;
+    let mut elem = c.find(Locator::Id("text-input")).await?;
+    assert_eq!(elem.prop("value").await?.unwrap(), "a");
+    Ok(())
+}
+
+async fn actions_mouse(mut c: Client, port: u16) -> Result<(), error::CmdError> {
+    let sample_url = sample_page_url(port);
+    c.goto(&sample_url).await?;
+
+    // Test pause.
+    let mouse_pause = MouseActions::new("mouse".to_string()).pause(Duration::from_secs(1));
+    let now = Instant::now();
+    c.perform_actions(vec![mouse_pause].into()).await?;
```

Comment:
With the change to `perform_actions`, I think this argument can just be `mouse_pause`, which'd be nice.